multiqc/utils/report.py

```diff
@@ -575,11 +574,14 @@ def save_htmlid(html_id, skiplint=False):
 
 
 def compress_json(data):
-    """Take a Python data object. Convert to JSON and compress using lzstring"""
+    """Take a Python data object. Convert to JSON and compress using gzip.
+    Represent in base64 format."""
     json_string = json.dumps(data).encode("utf-8", "ignore").decode("utf-8")
     json_string = sanitise_json(json_string)
-    x = lzstring.LZString()
-    return x.compressToBase64(json_string)
+    json_bytes = json_string.encode("utf-8")
+    json_gzip = gzip.compress(json_bytes, compresslevel=9, mtime=0)
```

Comment:
Could you add a comment in the code on why the `compresslevel` is set to the max? 

I guess there is a trade-off between the report size and the speed of loading, so wonder if you tried it with different levels? :)

Reply:
Doesn't make much difference, but I'll use all the default args to gzip.compress. Just pushed directly into the PR if that's okay